config/config.exs

```diff
@@ -0,0 +1,4 @@
+use Mix.Config
+
+config :ex_unit,
+  assert_receive_timeout: 1000
```

Comment:
`assert_receive_timeout: 1000` does not mean that we are always waiting for 1 second, but at most.